src/components/common/Spinner/spinner.test.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
-import { render } from 'test-utils';
+import { render } from '~/lib/test-utils';
 
-import Spinner from './Spinner';
```

Comment:
I'm still on the fence for this one. If I look at the button test it imports the Button with `~components`. Another option would be to `../Spinner` which is more explicit.

Do you have a preference for any of these options?